lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -84,6 +84,7 @@ class ReportUIFeatures {
     this._setupToolsButton();
     this._setupThirdPartyFilter();
     this._setUpCollapseDetailsAfterPrinting();
+    this._setUpQuickTabFocus();
```

Comment:
lol this `setUp` typo has survived a full 2 years https://github.com/GoogleChrome/lighthouse/pull/2000/files

Reply:
ahhhhh now I can't unsee.